### PR TITLE
Created docker compose file + script to run queries inside postgres container

### PR DESCRIPTION
**Pull request by adyach**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/318" target="_blank">ReviewNinja

File: database/nakadi/execute_db_scripes_in_container.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (v1ctor) [inline database/nakadi/execute_db_scripes_in_container.sh]: I think you have mistaken in file name

**Comment by v-stepanov** [comment]
@adyach was it approved already?

**Comment by v-stepanov** [comment]
@adyach for me everything worked out of the box. I just run `docker-compose up` and Nakadi started with all dependencies and I was able to make some requests to it. 
Great job.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      POSTGRES_PASSWORD: nakadi
+      POSTGRES_DB: local_nakadi_db
+
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+      - "2181:2181"
+
+  kafka:
+    image: wurstmeister/kafka
```
Comment (v-stepanov) [inline docker-compose.yml]: is it somehow possible to specify the version? (also interesting for postgres and zookeeper images)

File: database/nakadi/execute_db_scripts_in_container.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find /docker-entrypoint-initdb.d -type f -name "*.sql" -exec psql -U "$POSTGRES_USER" -d "$POSTGRES_DB" -a -f '{}' \;
```
Comment (v-stepanov) [inline database/nakadi/execute_db_scripts_in_container.sh]: Who executes this script

**Comment by v-stepanov** [comment]
@adyach 
As we have docker-compose now, I think it would be cool to use it in our build pipeline to get rid of our "Frankeschtein" local-storages image (kafka+zookeeper+postgres in one image). So that acceptance-tests and local run of Nakadi are performed using docker-compose approach instead of the old one.
What do you think?

File: docker-compose.yml (hunk 2 above)
Comment (adyach) [inline docker-compose.yml]: Yes, it is the same for docker images
image: wurstmeister/kafka:0.9.0.1
image: wurstmeister/zookeeper:3.4.6
image: postgres:9.5

File: database/nakadi/execute_db_scripts_in_container.sh (hunk 3 above)
Comment (adyach) [inline database/nakadi/execute_db_scripts_in_container.sh]: It is gonna be run by postgres container. For more info see section "How to extend this image": https://hub.docker.com/_/postgres/

**Comment by adyach** [comment]
Great idea! I will take it as my next task.

**Pull request merged**